tests/test_requests.py

```diff
@@ -33,6 +33,7 @@
 
 from .compat import StringIO, u
 from .utils import override_environ
+from requests.packages.urllib3.util import Timeout
```

Comment:
Ah, so the tests are failing because you've aliased this name (`Timeout`) with the name of an exception. Might be worth changing this to `from requests.packages.urllib3.util import Timeout as Urllib3Timeout` or something similar.